src/app/shared/components/message-bar/message-bar.component.css

```
@@ -0,0 +1,97 @@
+}
+
+.icon {
+  margin-right: 0.5rem;
+  border-radius: 2rem;
+  border: solid 1px #444444;
+  font-size: 13px;
+  display: flex;
+  flex-direction: row;
+  box-sizing: border-box;
+  justify-content: center;
+  place-content: center;
```

Biome (1.9.4)

[error] Unexpected shorthand property place-content after justify-content



(lint/suspicious/noShorthandPropertyOverrides)